newIDE/app/src/stories/componentStories/Profile/Subscription/SubscriptionDetails.stories.js

```diff
@@ -23,73 +25,156 @@ export default {
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
 );
 
 export const WithGoldSubscription = () => (
-  <SubscriptionDetails
-    subscription={subscriptionForGoldUser}
-    onManageSubscription={action('manage subscription')}
-    isManageSubscriptionLoading={false}
-  />
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={subscriptionForGoldUser}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
 );
 
 export const WithValidSilverRedemptionCodeSubscription = () => (
-  <SubscriptionDetails
-    subscription={silverSubscriptionWithRedemptionCode}
-    onManageSubscription={action('manage subscription')}
-    isManageSubscriptionLoading={false}
-  />
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={silverSubscriptionWithRedemptionCode}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
 );
 
 export const WithExpiredSilverRedemptionCodeSubscription = () => (
-  <SubscriptionDetails
-    subscription={silverSubscriptionWithExpiredRedemptionCode}
-    onManageSubscription={action('manage subscription')}
-    isManageSubscriptionLoading={false}
-  />
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={silverSubscriptionWithExpiredRedemptionCode}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
+);
+
+export const WithPurchaselyGoldSubscription = () => (
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={purchaselyGoldSubscription}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
 );
 
 export const WithLegacyIndieSubscription = () => (
-  <SubscriptionDetails
-    subscription={subscriptionForIndieUser}
-    onManageSubscription={action('manage subscription')}
-    isManageSubscriptionLoading={false}
-  />
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={subscriptionForIndieUser}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
 );
 
 export const WithLegacyProSubscription = () => (
-  <SubscriptionDetails
-    subscription={subscriptionForProUser}
-    onManageSubscription={action('manage subscription')}
-    isManageSubscriptionLoading={false}
-  />
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={subscriptionForProUser}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+    />
+  </AlertProvider>
+);
+
+export const WithNoSubscriptionOnMobile = () => (
+  <AlertProvider>
+    <SubscriptionDetails
+      subscription={noSubscription}
+      onManageSubscription={action('manage subscription')}
+      isManageSubscriptionLoading={false}
+      simulateNativeMobileApp
```

Comment:
I feel a bit uneasy with this prop, especially if we're testing a case of the mobile app, on the desktop app.
Do we need to keep this?